webapp/app.py

```diff
@@ -178,17 +178,39 @@ def _get_contributors():
     return contributors
 
 
-def _filter_contributors(contributors):
-    # Distinguish team_members from contributors
```

Comment:
Can we keep this comment about distinguishing team_members in the code please?

Reply:
Sure. I have added it back.